src/interface/cmdTlmServer.c

```diff
@@ -285,6 +341,63 @@ static bool cmdTlmServer_isCmdAuthorized(commandStatusStructure_t *pCmdStatus,
     return isCmdAuthorized;
 }
 
+// Send the telemetry and command status to the client.
+// Note: The data types of input and output are required for pthread_create().
+// The input needs to cast to the correct data type.
+static void *cmdTlmServer_sendTlmAndCmdStatus(void *pData) {
+
+    // Get the server information
+    serverInfo_t *pServerInfo = (serverInfo_t *)pData;
+
+    // Wait until begins to write the telemetry
+    while (!pServerInfo->isReadyTlm) {
+        sleep(1);
```

Comment:
1 second seems sluggish. If it's safe to do so (without bogging down the code), I suggest checking at something like 5-20 Hz. You could call `nanosleep(ts50)` if you defined ts50 a bit earlier.

Reply:
Updated the code in commit: 7473889. Thanks!